apps/web/src/components/buttons/SignUpAndInButtons.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactNode } from 'react';
+import { Button } from '@mui/material';
+
+interface ButtonProps {
+  onClick?: () => void;
+  text: string;
+  icon?: ReactNode;
+  type?: 'submit' | 'button';
+}
+
+const SignUpAndInButtons: React.FC<ButtonProps> = ({
+  text,
+  icon,
+  onClick,
+  type,
+}) => {
+  return (
+    <Button
+      variant="outlined"
+      fullWidth
+      sx={{
+        borderColor: 'rgb(97, 137, 47)', // Yeşil sınır rengi
+        color: 'rgb(97, 137, 47)', // Yeşil metin rengi
```

Comment:
@cihanbagriyanik @pierrelstan we don't have a theme for all these common colors?

Reply:
@dadiorchen , @pierrelstan 
I took the color code for Greenstand green from Figma and provided a visual representation based on what I saw there. If you have another suggestion, please feel free to share a visual reference, and I can make those adjustments. If you'd prefer a custom color from MUI or any other color instead of green, I can implement that as well—just let me know!